.github/ansible/deploy.yaml

```diff
@@ -0,0 +1,10 @@
+- hosts: all
+  remote_user: ghivr
+  gather_facts: false
+
+  tasks:
+  - name: pull an image
+    docker_image:
+      name: hjacobs/tiny-docker-http-test
```

Comment:
Is this to test it out? I have played around with the container registry on Github and we can connect packaging to the repository if need be.